samples/hosting/generic-host/Core_7/GenericHost/GenericHost.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+    <LangVersion>7.1</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.*" />
```

Comment:
why is this reference needed? Hosting also has a transitive dependency on the MS.Extensions.DI package

Reply:
Yeah it was auto added for some reason. Removed. Good catch